Source/WebCore/Modules/applepay/ApplePaySessionPaymentRequest.h

```diff
@@ -167,7 +167,90 @@ class ApplePaySessionPaymentRequest {
     const std::optional<ApplePayLaterAvailability>& applePayLaterAvailability() const { return m_applePayLaterAvailability; }
     void setApplePayLaterAvailability(const std::optional<ApplePayLaterAvailability>& applePayLaterAvailability) { m_applePayLaterAvailability = applePayLaterAvailability; }
 #endif
-    
+
+    ApplePaySessionPaymentRequest(String&& countryCode,
+        String&& currencyCode,
+        struct ContactFields&& requiredBillingContactFields,
+        PaymentContact&& billingContact,
+        ContactFields&& requiredShippingContactFields,
+        PaymentContact&& shippingContact,
+        Vector<String>&& supportedNetworks,
+        struct MerchantCapabilities&& merchantCapabilities,
+        ApplePaySessionPaymentRequestShippingType&& shippingType,
+        Vector<ApplePayShippingMethod>&& shippingMethods,
+        Vector<ApplePayLineItem>&& lineItems,
+        ApplePayLineItem&& total,
+        String applicationData,
+        Vector<String>&& supportedCountries,
+        Requester&& requester,
+#if HAVE(PASSKIT_INSTALLMENTS)
+        PaymentInstallmentConfiguration&& installmentConfiguration,
```

Comment:
commas before the conditional parameters instead of after.  That way it'll compile successfully with or without the feature on.

Reply:
Fixed